src/Initialization.cpp

```diff
@@ -157,6 +196,7 @@ namespace Initialization
                     number = (unsigned int) rint( ((Real) Ndaywork[to]) /
                              ((Real) WG_size * (Start[to+1] - Start[to])) );
 
+                    /* QDG ?? */
                     work_nborhood_ptr[ip]=4*(amrex::Random_int(4, engine))+nborhood_ptr[ip];
```

Comment:
@atmyers @tannguyen153 Why is the work neighborhood being set this way?

Reply:
The idea is that one number stores both 4 unique work neighborhoods and 4 unique home neighborhoods. From this value, you can recover the home neighbor by taking `% 4` and getting the remainder, or you can compute `/ 4` to get a work cluster that is between 0 and 3.